public/ipython/tree/js/clusterlist.js

```diff
@@ -245,6 +254,23 @@ define([
           keyboard_manager: that.keyboard_manager
         });
     });
+
+    delete_button.click(function (e) {
+        var settings = {
+            processData : false,
+            data : "name="+that.data.name,
+            cache : false,
+            type : "GET",
```

Comment:
would it be better to used `DELETE` here? 
So (below) we can stick with the `cluster` path